debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseSchema.java

```diff
@@ -33,15 +34,15 @@
     private final SchemasByTableId schemasByTableId;
     private final Tables tables;
 
-    protected RelationalDatabaseSchema(String serverName, TopicSelector<TableId> topicSelector,
-            Predicate<TableId> tableFilter, Predicate<ColumnId> columnFilter, ColumnMappers columnMappers,
-            TableSchemaBuilder schemaBuilder, boolean tableIdCaseInsensitive) {
+    protected RelationalDatabaseSchema(Configuration config, String serverName, TopicSelector<TableId> topicSelector,
```

Comment:
`serverName` is a part of `Configuration` so it should not be passed separately

Reply:
Ok, changed that to pass `CommonConnectorConfig` (which exposes `getLogicalName()`) instead.